src/main/app/fees/feesClient.ts

```diff
@@ -39,7 +39,8 @@ export default class FeesClient {
       throw new Error('Fee channel is required')
     }
     ClaimValidator.claimAmount(claimValue)
-    const feeUri: string = `${feesUrl}/fees-register/fees/lookup?service=${service}&jurisdiction1=${jurisdiction1}&jurisdiction2=${jurisdiction2}&channel=${paperChannel}&event=${issueFeeEvent}&amount_or_volume=${claimValue}`
+    const keyword: string = 'MoneyClaim HearingSmallClaims counter-claim'
```

Comment:
Why are we passing all keywords at the same time. In the excel it is given we have to pass 
"Civil Court fees " should be passing keyword as "MoneyClaim"
"Hearing fee" should be passing keyword as "HearingSmallClaims"
"Counter Claim" should be passing keyword as "counter-claim".
We should pass keyword accordingly in the fees url not all keywords at the same time